src/application.ts

```diff
@@ -44,19 +44,20 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
     // Requirements; for Production these are stored in GitHub repo secrets
     //
     // For Dev, you need to pass them in via .env file
-    const environment: string = process.env.NODE_ENV ?? 'production'
+    const environment: string = process.env.NODE_ENV || 'production'
 
     logger.log('info', 'Environment: ' + environment)
 
-    const dispatchURL: string = process.env.DISPATCH_URL ?? ''
-    const altruists: string = process.env.ALTRUISTS ?? ''
-    const clientPrivateKey: string = process.env.GATEWAY_CLIENT_PRIVATE_KEY ?? ''
-    const clientPassphrase: string = process.env.GATEWAY_CLIENT_PASSPHRASE ?? ''
-    const pocketSessionBlockFrequency: string = process.env.POCKET_SESSION_BLOCK_FREQUENCY ?? ''
-    const pocketBlockTime: string = process.env.POCKET_BLOCK_TIME ?? ''
-    const relayRetries: string = process.env.POCKET_RELAY_RETRIES ?? ''
-    const databaseEncryptionKey: string = process.env.DATABASE_ENCRYPTION_KEY ?? ''
-    const aatPlan = process.env.AAT_PLAN ?? AatPlans.PREMIUM
+    const dispatchURL: string = process.env.DISPATCH_URL || ''
+    const altruists: string = process.env.ALTRUISTS || ''
+    const clientPrivateKey: string = process.env.GATEWAY_CLIENT_PRIVATE_KEY || ''
+    const clientPassphrase: string = process.env.GATEWAY_CLIENT_PASSPHRASE || ''
+    const pocketSessionBlockFrequency: string = process.env.POCKET_SESSION_BLOCK_FREQUENCY || ''
+    const pocketBlockTime: string = process.env.POCKET_BLOCK_TIME || ''
+    const relayRetries: string = process.env.POCKET_RELAY_RETRIES || ''
+    const databaseEncryptionKey: string = process.env.DATABASE_ENCRYPTION_KEY || ''
+    const defaultSyncAllowance: number = parseInt(process.env.DEFAULT_SYNC_ALLOWANCE) || -1
```

Comment:
is 0 a valid value or must always be over 0? @nymd

Reply:
0 is valid though it would be very restrictive. It should be allowed. But not blank or undefined.

Reply:
all right, then this is ready